enos/modules/backend_test_servers/main.tf

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+terraform {
+  required_providers {
+    enos = {
+      source = "registry.terraform.io/hashicorp-forge/enos"
+    }
+  }
+}
+
+locals {
+  distro = var.distro
+  ldap_server = {
+    domain     = "enos.com"
+    org        = "hashicorp"
+    admin_pw   = "password1"
+    version    = var.ldap_version
+    port       = "389"
```

Comment:
Forward thinking here: we probably ought to make all of these variables with default values and then set them here so we can output is as state.